lib/stackify/env_details.rb

```diff
@@ -11,6 +11,27 @@ class EnvDetails
 
 
     def initialize
```

Comment:
consolidate set_rails_info and initialize logic into a single function.  It should be callable and if rails Rails::Info exists - it uses it.

Reply:
Hello Darin, 

I just made an update based from your comment.